### ISSUE TITLE
Hyperspy not installing on Anaconda Python 3.6 due to package conflicts

### ISSUE THREAD

**Issue by thomasaarholt** [hyperspy/hyperspy#1499]
It appears that despite what was commented in #1456, Hyperspy does not successfully install in python 3.6 (ref maillist issues). I think it's important to fix before releasing 1.2.

**Comment by thomasaarholt** [comment]
Just realised I have mixed this up. It should correctly install from github, whilst not install from 1.12 on conda-forge.

**Comment by thomasaarholt** [comment]
Actually, this worked fine for me:
```bash
conda create --name test_environment_name python=3.6
source activate test_environment_name
conda config --add channels conda-forge
conda install hyperspy
```

**Comment by to266** [comment]
This works in miniconda and/or new environments because `anaconda-navigator` and `anaconda` are in conflict with the `pyqt=4` and `python=3.6`.

If you want to use the root (main) environment, just uninstall the anaconda-navigator before installing hyperspy (you probably don't need it to begin with.. I certainly don't, have no clue what it does!)
```bash
conda uninstall anaconda-navigator -y
conda install -y -c conda-forge hyperspy
```

**Comment by francisco-dlp** [comment]
Installing in Anaconda is harder than it should be due to this issue. I've reopen it so that it's easier to find by anybody experiencing it. There are two paths to fixing this:

1. Do nothing and wait untils traitsui adds support for Qt5.
2. Make tratisui optional.

As chances of 1. happening anytime soon are low, 2. may be the way to go.

**Comment by sem-geologist** [comment]
or drop traits completely...

**Comment by to266** [comment]
@sem-geologist we want to do this for a while now, however can't justify spending the time. Feel free to tackle it, everyone will be happy.

**Comment by francisco-dlp** [comment]
Fixed in #1544
